maintenance/wikia/jwplayerattribution/migrateJWAttributionToDashboard.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+/**
+ * Migrate JW videos attribution from WF to JW Dashboard
+ *
+ */
+require_once( dirname( __FILE__ ) . '/../../Maintenance.php' );
+require_once('jwplayer-api-kit/botr/api.php');
+
+class MigrateJWAttributionToDashboard extends Maintenance {
+
+	protected $dryRun = true;
+
+	protected $map = [];
+
+	public function __construct() {
+		parent::__construct();
+		$this->mDescription = "Migrate JW videos attribution from WF to JW Dashboard";
+		$this->addOption( 'dry-run', 'Dry run mode', false, false, 'd' );
+
+		$this->jwApi = new BotrAPI('key', 'secret');
+
+	}
+
+	public function execute() {
+		$this->dryRun = $this->hasOption( 'dry-run' );
+		$this->output( "Start \n" );
+
+		$this->doUpdate( 1689, 'wgArticleVideoFeaturedVideos' );
+		$this->doUpdate( 1749, 'wgArticleVideoFeaturedVideos2' );
+	}
+
+	private function doUpdate( $varId, $varName ) {
+		// Get all wikis with var set
+		$wikisWithVar = ( WikiFactory::getListOfWikisWithVar( $varId, 'array', '!=', [] ) );
+
+		$this->output( 'Found ' . count( $wikisWithVar ) . "wikis with " . $varName . " config\n" );
+
+
+		// Iterate over all wikis that have the var set
+		foreach ( $wikisWithVar as $wikiId => $wiki ) {
```

Comment:
I know this way is easier now since it is copy-paste, but in the future can we just run maintenance script like that: https://wikia-inc.atlassian.net/wiki/spaces/ONE/pages/35724059/Engineering+Onboarding+How+to+run+maintenance+script
which by default iterates on each wiki, so the  only thing that we'd need to do in this script would be checking if wiki has FV variable and migrate if yes